src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -403,22 +403,88 @@ private static Version GetMSBuildVersionInPath()
         /// <returns>The msbuild directory.</returns>
         public static string GetMsbuildDirectory(string userVersion, IConsole console)
         {
-            List<Toolset> installedToolsets;
-            using (var projectCollection = new ProjectCollection())
+            // Try to find msbuild for mono from hard code path.
```

Comment:
Add comment about Mono bug, which means Mono always tells use we are on Unix (even if we really are Mac OS X). The constants below mentioning "Mac" are correct.